src/Graby.php

```diff
@@ -167,11 +167,11 @@ public function getConfig(string $key)
      *
      * @return array With keys html, title, url & summary
      */
-    public function fetchContent(string $url): array
+    public function fetchContent(string $url, string $prefetchedContent = null): array
```

Comment:
Passing a content to a method called `fetchContent` still feels a bit iffy to me, I would prefer a separate `cleanContent` method. But then, the code has been a pile of megafunctions playing fast and loose with SRP since it was called Full-Text RSS, and I do not see a way to separate it out cleanly without a huge refactoring.

Reply:
Separating fetch and extraction logics from `doFetchContent()` may be difficult because of the multipage fetcher.

Another approach instead of passing content in the `fetchContent()` could be to have a separate setter like `setContentAsPrefetched($content)` that we would call before calling `fetchContent()`.

Reply:
Great suggestion @Kdecherf  with `setContentAsPrefetched`. I prefer it. Can you make the change?

Reply:
@j0k3r done, I think we're good to make another review